docs: add missing hotkeys (#38)
autowarefoundation/AWSIM: docs/ProjectGuide/HotkeyList/index.md

|Right arrow| Steering (Right). |
|Down arrow| Breaking. |

> W,A,S,D is also used for vehicle control, just like arrow keys.

Reviewer: ```suggestion
W,A,S,D keys can also be used to control vehicles, similar to the arrow keys.
```
